skyportal/tests/api/test_candidates.py

```diff
@@ -1503,49 +1505,3 @@ def test_candidate_list_pagination(
         token=view_only_token,
     )
     assert status == 400
-    assert "Page number out of range" in data["message"]
```

Comment:
Why is this line deleted...?

Reply:
Because I don't know how copy and paste works.